Makefile

```diff
@@ -49,6 +49,7 @@ validation_client_run: validation-clean
 	VALIDATION_CLIENT_INPUT_PATH=validation-tmp/outputs/output_1 VALIDATION_CLIENT_OUTPUT_PATH=validation-tmp/inputs/input_2 python validation-client/client.py
 
 docker_compose: validation-clean
+	docker-compose down
```

Comment:
Minor: `docker-compose` command I think has been deprecated in favor of `docker compose`. I am not sure if this is a problem for this specific service (I guess the CI successfully builds it anyway).

Reply:
Completely true, I was mirroring what was already there in the original Makefile. But I've replaced now all docker-compose with docker compose.